app/src/main/res/xml/home_preferences.xml

```diff
@@ -29,6 +29,11 @@
         android:title="@string/customize_toggle_pocket"
         app:isPreferenceVisible="false" />
 
+    <androidx.preference.Preference
+        android:key="@string/pref_key_wallpapers"
+        android:title="@string/customize_wallpapers"
+        app:isPreferenceVisible="true" />
```

Comment:
Should this be visible by default already?

Reply:
If this is behind a feature flag and only shown for a subset of our users, we tend to default to false to avoid showing the preference and hiding it immediately (causing a flash).